src/arguments.js

```diff
@@ -15,14 +15,28 @@ const getArgumentObject = (object, context) => {
         shortName: getPropertyName(object.arg, context.regexp.argument.short),
         defaultValue: getDefaultValue(
             object.description, context.regexp.defaultValue),
-        description: object.description.trim(),
+        description: unifyDescription(object.description),
     };
     argument.isFlag = isDefaultBollean(argument.defaultValue)
         && !isPropertyTyped(object.arg);
 
     return argument;
 };
 
+/**
+ * Unify description to one standard
+ * @param {string} description - argument description
+ * @return {string} description - unified description
+ */
+const unifyDescription = (description) => {
+    description = description.trim();
+    description = description.charAt(0).toUpperCase() + description.slice(1);
+    if (description.charAt(description.length - 1) === '.') {
```

Comment:
I'm not sure but we can use ternary operator for this

Reply:
@svoboda-rabstvo done.